ballerina/tests/client_test.bal

```
@@ -0,0 +1,629 @@
+import ballerina/test;
+import ballerina/sql;
+import ballerina/io;
+
+@test:Config {}
+function testBasicClientCreation() returns error? {
+    // Test basic client creation with username/password only
+    Client|error basicClient = trap new(TEST_USER, TEST_PASSWORD);

```

```suggestion
    Client|sql:Error basicClient = new(TEST_USER, TEST_PASSWORD);

```

---

you don't need to trap, as it returns an `sql:Error` in the init APIs

---

You can also return the error like below; the test will fail if it returns an error

Client basicClient = check new(TEST_USER, TEST_PASSWORD);

---

please check all the init test cases

---

fixed with [90225b2](https://github.com/ballerina-platform/module-ballerinax-cdata.connect/pull/26/commits/90225b2c38dbf67a5d18a8fcee105b38c2af75e7)